src/game_inventory.cpp

```diff
@@ -534,22 +534,36 @@ class comestible_inventory_preset : public inventory_selector_preset
                 return string_format( _( "%.2f%s" ), converted_volume, volume_units_abbr() );
             }, _( "VOLUME" ) );
 
+            // Title of this cell. Defined here in order to preserve proper padding and alignment of values in the lambda.
+            constexpr char *nourishment_title = static_cast<char *>( "NOURISHMENT" );
```

Comment:
Might be able to be static too, I'm not sure if there's still a problem with making translated strings static.
```suggestion
            const std::string nourishment_title = _( "NOURISHMENT" );
```

Reply:
Can you make it "Nourishment" for the localized string and just uppercase it for the column header? I'm getting stupid ideas ;)

Reply:
That *is* a stupid idea, we recently had a crash/garbage characters issue on some OSes because uppercasing is apparently borked for some languages (Cyrilic, CJK) on some OSes.

Reply:
There are some concerns about uppercasing unicode strings (#44288), but hopefully what I just linked resolved that, so possibly.

Reply:
Yeah, no problem adding a secondary, properly cased resource string then.